pkg/install/0-installstorage.go

```diff
@@ -40,29 +34,7 @@ func (i *Installer) createDNS(ctx context.Context) error {
 	return i.dns.Create(ctx, i.doc.OpenShiftCluster)
 }
 
-func (i *Installer) installStorage(ctx context.Context, installConfig *installconfig.InstallConfig, platformCreds *installconfig.PlatformCreds, image *releaseimage.Image) error {
-	clusterID := &installconfig.ClusterID{
```

Comment:
I like the ideas here - I'm currently working on https://github.com/Azure/ARO-RP/issues/106 in https://github.com/Azure/ARO-RP/compare/master...nilsanderselde:refactor-installer, so I'm also looking at possibly refactoring this and breaking stuff out to be testable. It's difficult to do without a lot of churn, though :/

To avoid duplicated effort, I will work on unit testing existing install steps for now until I see what becomes of this PR.

If the new function(s) you're moving this into are unit tested, that would dovetail nicely into what I'm doing.

Reply:
@nilsanderselde I wasn't aware that the cleanup was a WIP, but I was advised to not do any restructuring in my PR, so go ahead with your work.